deploy/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Builder image
+FROM golang:1.16 as builder
+
+# Workaround because migration tool is private repo
+ARG SSH_PRIVATE_KEY
+RUN mkdir /root/.ssh/ && \
+    echo "${SSH_PRIVATE_KEY}" > /root/.ssh/id_rsa && \
```

Comment:
currently building this using
```bash
SSH_PRIVATE_KEY="$(cat ~/.ssh/id_rsa)" make build-image push-image
```
the ssh key is only present in the interim builder image and doesn't get to the runnable image